code/file_handling.py

```diff
@@ -43,16 +43,20 @@ def singleplane_session_data_files(input_path: Union[str, Path]):
 
     # There should be exactly one subfolder inside
     # the single-plane parent folder
+    img_path = ""
     subfolders = [f for f in input_path.iterdir() if f.is_dir()]
     if len(subfolders) == 1:
         singleplane_data_path = subfolders[0]
     if len(subfolders) == 0:
         singleplane_data_path = input_path
-    if len(subfolders) != 1 and len(subfolders) != 0:
-        raise ValueError(
-            f"Expected exactly one or zero subfolder in {input_path},  \
-              found {len(subfolders)}"
-        )
+
+    for subfolder in subfolders:
+        if "capsule" in subfolder.name:
+            singleplane_data_path = subfolder
+        # newer assets have the motion_correcton in its own folder
+        if "MOp2_3_0" in subfolder.name:
```

Comment:
Does this name have to be hardcoded - if so, I will create an issue to fix this and pull from the metadata

Reply:
this is a confusing one because of a metadata mismatch. I will lay out the work to fix it